spec.html

```diff
@@ -4,7 +4,7 @@
 title: Numeric separators
 status: proposal
 stage: 1
-location: https://github.com/samuelgoto/proposal-numeric-separator
+location: https://github.com/tc39/proposal-numeric-separator
 copyright: false
 contributors: Sam Goto
```

Comment:
i don't know exactly what are the semantics/visibility of this file and this line specifically, but if you are ok, i'd like to include your name in whichever list my name appears in: you wrote a significant part of the spec text and i'd like to give attribution where attribution is due.